index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Personal Portfolio</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <nav class="nav-bar">
+            <a class="logo" href="#">Snoplezik</a>
+            <a class="hamburger" href="#"><div></div><div></div><div></div></a>
+        </nav>
+        <header class="headline">
+            <img class="background" src="images/Illustration-header.png" alt="">
```

Comment:
- [ ] kindly consider fixing your background so the white space in the right is removed, 

![Screenshot_20220701_043009](https://user-images.githubusercontent.com/25227022/176818448-b28fa428-f6fa-453e-84eb-007c31afe963.png)